apollo-router/src/plugins/telemetry/metrics/apollo/snapshots/apollo_router__plugins__telemetry__metrics__apollo__test__apollo_metrics_multiple_operations.snap

```diff
@@ -1,5 +1,60 @@
 ---
-source: apollo-router/src/plugins/telemetry/metrics/apollo.rs
+source: apollo-router/src/plugins/telemetry/metrics/apollo/mod.rs
 expression: results
 ---
-[]
+[
```

Comment:
simon and i talked about this yesterday a bit - it's really strange these snapshots were _empty_ before. as far as we could tell they were to test failures, but no failures were recorded? simon's found a bug with us not using [spec errors](https://github.com/apollographql/router/pull/5948/files/f0c0a67d81d3c020cd347cc4c8330b57905f5619#diff-4927e3b4cecf4baec251ae4b7e1ed38814b22f3a659c20c7b7cf6b864fe79aecR54-R58) for unknown operations even though the error was defined. these snapshots now actually return something because we actually use those errors.

Reply:
I meant to but forgot to ask @tninesling who added them in https://github.com/apollographql/router/pull/5443: was it intentional to write a test that looks like it’s generating some metrics but asserts nothing is generated?

Reply:
That snapshot file was renamed in that change, but I didn't add it originally. From looking at the test, I can't tell exactly what the output should be, but I would be surprised if it was meant to be empty.